recipes/dedalus/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "dedalus" %}
+{% set version = "2.1905" %}
+
+{% set mpi_prefix = "mpi_" + mpi %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 92d2a89c5516902848673f0f00610ebfb993d269bb8ec7c248d9313854d56eaf
+
+build:
+  number: 0
+  skip: True  # [win or py>37]
+  script:
+    - export FFTW_PATH=$PREFIX
+    - export FFTW_INCLUDE_PATH=$PREFIX/include
+    - export FFTW_LIBRARY_PATH=$PREFIX/lib
+    - export MPI_PATH=$PREFIX
+    - export MPI_INCLUDE_PATH=$PREFIX/include
+    - export MPI_LIBRARY_PATH=$PREFIX/lib
+    - {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - pip
+    - python
+    - setuptools
+    - cython
+    - mpi4py
+    - numpy
+    - fftw
+    - fftw * {{ mpi_prefix }}_*
+
+  run:
+    - python
+    - docopt
+    - h5py
+    - fftw
+    - fftw * {{ mpi_prefix }}_*
+    - matplotlib
```

Comment:
Should be `matplotlib-base`

Reply:
@scopatz these changes have been made. thanks!